src/main/webapp/i18n/de/exam.json

```
@@ -158,6 +192,7 @@
             "exercisePoints": "Punkte",
             "exerciseName": "Name",
             "noStudentExam": "Du bist nicht für die Klausur {{ examTitle }} angemeldet. Bitte wende dich an die entsprechende Lehrkraft.",
+            "noFurtherAttempts": "Derzeit sind keine weiteren Versuche für die Testklausur {{ examTitle }} möglich. Um deine Abgabe zu betrachten, navigiere zur Exam Seite ",
```

```suggestion
            "noFurtherAttempts": "Derzeit sind keine weiteren Versuche für die Testklausur {{ examTitle }} möglich. Um deine Abgabe zu betrachten, navigiere zur Klausurseite ",
```
I guess we want to use the German word word exam :D